sdks/python/apache_beam/dataframe/frame_base.py

```diff
@@ -588,15 +622,23 @@ def format_section(header):
   return wrap
 
 
-def populate_defaults(base_type):
+def populate_defaults(base_type, removed_method=False, removed_args=None):
   """Populate default values for keyword arguments in decorated function.
 
   When applied to a function, this decorator creates a new function
   with default values for all keyword arguments, based on the default values
   for the identically-named method on `base_type`.
 
-  For internal use only. No backwards compatibility guarantees."""
+  removed_method used in cases where a method has been removed in a later
```

Comment:
nit: let's split line for readability (or use   `Args:` format),
```
  removed_method used in cases where a method has been removed in a later
  version of Pandas.
  removed_args used in cases where a method has had arguments removed in a
  later version of Pandas.
```

Reply:
Done.